service/src/main/java/uk/gov/hmcts/reform/fpl/request/AuthorizationConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.request;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.context.annotation.RequestScope;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Configuration
+public class AuthorizationConfiguration {
+
+    @Bean
+    @RequestScope
+    public RequestData authorisation(HttpServletRequest request) {
```

Comment:
class and method is about authorisation, but returns quite generic RequestData

Reply:
I think this might be expanded to include `user-id`

Reply:
Refactored to what we discussed, thanks